packages/SwingSet/src/kernel/state/storageWrapper.js

```diff
@@ -64,6 +83,18 @@ export function guardStorage(hostStorage) {
   return harden({ has, getKeys, get, set, delete: del });
 }
 
+/**
+ * Create and return a crank buffer, which wraps a storage object with logic
+ * that buffers any mutations until told to commit them.
+ *
+ * @param storage  The storage object that this crank buffer will be based on.
+ *
+ * @return an object {
+ *   crankBuffer,  // crank buffer as described, wrapping `storage`
+ *   commitCrank,  // capfn, saves buffered mutations to `storage` when invoked
```

Comment:
what does `capfn` mean?

Reply:
That was an on the fly abbreviation for "capability function", i.e., an ocap expressed as a function, introduced solely in the interest of keeping the comment short enough to prevent the line from wrapping.  But if it wasn't obvious, the probably it should be rewritten.